install/wipe_ncn_disks_for_reinstallation.md

```diff
@@ -178,14 +208,18 @@ RAIDs, zeroing the disks, and then wiping the disks and RAIDs.
 
     1. Master nodes.
 
+        Stop the etcd service on the master node before unmounting /var/lib/etcd
+
         ```bash
-        ncn-m# umount -v /var/lib/etcd /var/lib/sdu
+        ncn-m# systemctl stop etcd.service
+        ncn-m# umount -v /run/lib-etcd /var/lib/etcd /var/lib/sdu
```

Comment:
`umount` the overlayFS first to properly undo layering.

This might work as-is, but it would be proper to tell the kernel the overlayFS is going down first to remove doubt that the persistent storage was yanked out early.

```bash
ncn-m# umount -v /var/lib/etcd /run/lib-etcd /var/lib/sdu
```

Reply:
In hindsight, this is just wiping the magic bit.

The order doesn't matter, if this was actually erasing or writing zeros then I'd say the order would matter.

This can be resolved.